### PR TITLE
Add display properties to attribute table

### LINKED ISSUE #1111: [FEATURE] Add hide column, set column width and autosize to
attribute table header right click menu
Original commit: https://github.com/qgis/QGIS/commit/384a280f7131820d80bee027d446c1da568742d2 by nyalldawson

Unfortunately this naughty coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by yjacolin**
* hide columns, organize, set with vs autosize
* better index entry

Fix #1111

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/attribute_table.rst [hunk 1]
```diff
@@ -83,7 +83,17 @@ Calculator` (see :ref:`calculate_fields_values`).
    option in :menuselection:`Settings --> Options --> Data Sources` menu.
 
 
-.. index:: Attribute Table Selection
+.. index::
+   pair: Attributes; Columns
+
+One can hide columns or change their width (either settings the width or with
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/attribute_table.rst]: setting

File: source/docs/user_manual/working_with_vector/attribute_table.rst [hunk 2]
```diff
@@ -83,7 +83,17 @@ Calculator` (see :ref:`calculate_fields_values`).
...
+the :guilabel:`Autosize` settings) by right-clicking in the table header. To
+change several columns behaviour in once or display again the column, choose
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/attribute_table.rst]: `behavior` (to be US-compliant)
`at once or unhide a column`

File: source/docs/user_manual/working_with_vector/attribute_table.rst [hunk 3]
```diff
@@ -83,7 +83,17 @@ Calculator` (see :ref:`calculate_fields_values`).
...
+:guilabel:`Organize columns ...`. Finally one can choose to sort the rows with
+the :guilabel:`sort` and write an expression.
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/attribute_table.rst]: An example of sorting with multicolumns can be of interest here, I think (may be with `concat`). Not sure it'd be that obvious for users

File: source/docs/user_manual/working_with_vector/attribute_table.rst [hunk 4]
```diff
@@ -25,8 +25,8 @@ in the Attributes toolbar.
 
 This will open a new window that displays the feature attributes for the
 layer (figure_attributes_1_). The number of features and the number of
-selected features are shown in the attribute table title. Columm widths can be 
-changed by dragging the boundary on the right of the column heading, resized column 
+selected features are shown in the attribute table title.  Columm widths can be
```
Comment (SrNetoChan) [inline source/docs/user_manual/working_with_vector/attribute_table.rst]: Is this extra space needed?

**Comment by SrNetoChan** [comment]
LGTM

File: source/docs/user_manual/working_with_vector/attribute_table.rst [hunk 5]
```diff
@@ -83,7 +83,18 @@ Calculator` (see :ref:`calculate_fields_values`).
...
 
 
-.. index:: Attribute Table Selection
+.. index::
+   pair: Attributes; Columns
+
+One can hide columns or change their width (either setting the width or with
+the :guilabel:`Autosize` settings) by right-clicking in the table header. To
+change several columns behavior at once or unhide a column, choose
+:guilabel:`Organize columns ...`. Finally one can choose to sort the rows with
```
Comment (SrNetoChan) [inline source/docs/user_manual/working_with_vector/attribute_table.rst]: missing comma after 

> Finally, one can

**Comment by yjacolin** [comment]
Need more work on change the order of the column, and the possibility to add "action" column (via organizing column menu)

File: source/docs/user_manual/working_with_vector/attribute_table.rst [hunk 6]
```diff
@@ -83,7 +83,20 @@ Calculator` (see :ref:`calculate_fields_values`).
...
 
-.. index:: Attribute Table Selection
+.. index::
+   pair: Attributes; Columns
+
+You can hide columns or change their width (either setting the width or with
+the :guilabel:`Autosize` setting) by right-clicking in the table header. To
+change several columns behavior at once, unhide a column or change the order of
+the columns, choose :guilabel:`Organize columns ...`. In this dialog, you can
+also add a new :guilabel:`Actions` column that add a dropdown list of actions
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/attribute_table.rst]: that adds

File: source/docs/user_manual/working_with_vector/attribute_table.rst [hunk 7]
```diff
@@ -83,7 +83,20 @@ Calculator` (see :ref:`calculate_fields_values`).
...
+for each rows. Finally, one can choose to sort the rows with the
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_vector/attribute_table.rst]: each row

**Comment by DelazJ** [comment]
lgtm

**Pull request merged**